src/packages/sdk/src/getAllVersions.ts

```diff
@@ -0,0 +1,35 @@
+import fetch from 'node-fetch'
+import fs from 'fs'
+import path from 'path'
+export { engineVersions } from './versions'
+
+async function getAllVersions() {
+  const url = `https://registry.npmjs.org/@prisma/cli`
+  const pkg = await fetch(url).then(r => r.json())
```

Comment:
We should use `getProxyAgent` here I think

Reply:
Not necessary, as this code is not being executed on any user machine.

Reply:
Indeed 👍